tests/menus/test_frequently_used_context_menu.py

```diff
@@ -0,0 +1,120 @@
+import platform
+from time import sleep
+
+import pytest
+from selenium.webdriver import Firefox
+
+from modules.browser_object import ContextMenu, Navigation
+from modules.page_object import ExamplePage
+from modules.util import BrowserActions, Utilities
+
+
+@pytest.mark.unstable
+def test_save_page_as(driver: Firefox):
+    """
+    C2637623.1: save page as
+    """
+    try:
+        from pynput.keyboard import Key
+    except ModuleNotFoundError:
+        pytest.skip("Could not load pynput")
+    # create objects
+    context_menu = ContextMenu(driver)
+    driver.get("https://example.com")
+    example_page = ExamplePage(driver)
+    nav = Navigation(driver)
+    util = Utilities()
+    ba = BrowserActions(driver)
+
+    # right click something that is not a hyperlink
+    title_header = example_page.get_element("title-header")
+    context_menu.context_click_element(title_header)
+
+    with driver.context(driver.CONTEXT_CHROME):
+        save_page_as = context_menu.get_context_item("context-menu-save-page-as")
+        save_page_as.click()
+
+    downloads_button = nav.get_download_button()
+
+    # short sleep to ensure menu is shown
+    sleep(0.5)
+
+    # perform key presses to save the file
+    this_platform = platform.system()
+    if this_platform == "Linux":
+        ba.controller.press(Key.alt)
+        ba.controller.press(Key.tab)
+        ba.controller.release(Key.tab)
+        ba.controller.release(Key.alt)
+
+        ba.controller.press(Key.alt)
+        ba.controller.press(Key.tab)
+        ba.controller.release(Key.tab)
+        ba.controller.release(Key.alt)
+
+        ba.key_press_release(Key.tab)
+
+        ba.key_press_release(Key.tab)
+
+    # Press and release the Enter key
+    ba.key_press_release(Key.enter)
+
+    # Wait for the animation to complete
+    with driver.context(driver.CONTEXT_CHROME):
+        nav.wait_for_download_animation_finish(downloads_button)
```

Comment:
Same as above, is there a way to make this method change the context for us?

Reply:
Got it!